test/unit/block/shared/src/BlockSharedMemDyn.cpp

```diff
@@ -37,7 +37,7 @@
 //#############################################################################
 //!
 //#############################################################################
-class BlockSharedMemDyn
+class BlockSharedMemDynTestKernel
```

Comment:
Because I wrote a new test kernel, looked for the current naming convention and found that this was the only test kernel that was named inconsistently :smile: